packages/slice-machine/server/src/api/state.ts

```diff
@@ -68,7 +68,7 @@ export default async function handler() {
         Math.floor(newTokenResponse.status / 100) === 2
       ) {
         const newtToken = await newTokenResponse.text();
```

Comment:
little typo here

Reply:
? it's not even a change, I don't see the typo xD